src/components/OfferListTable.js

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from 'react';
+import { getitems, getitembyid, edititem, deleteitem } from '../APIs/ApiOffer';
+
+export default function OfferListTable({ isFilter, searchfilter, selectcategory }) {
```

Comment:
don't use isFilter param instead use seearchFilter and selectCategory

Reply:
Done